requirements_gpu.txt

```diff
@@ -0,0 +1 @@
+cupy
```

Comment:
This was the package being installed before, unless you changed it intentionally.
```suggestion
cupy-cuda102
```

Reply:
Good catch!